packages/core/contracts/optimistic-assertor/implementation/OptimisticAssertor.sol

```diff
@@ -125,12 +125,14 @@ contract OptimisticAssertor is Lockable, OptimisticAssertorInterface, Ownable {
 
     function getAssertion(bytes32 assertionId) public view returns (bool) {
         Assertion memory assertion = assertions[assertionId];
+        // Return early if not using answer from resolved dispute.
+        if (assertion.disputer != address(0) && !assertion.useDisputeResolution) return false;
```

Comment:
If a user calls the public getter of assertions like this `OA.assertions(assertionID).resolutionValue` and the assertion was a disputed one with `assertion.useDisputeResolution = false` he could get a wrong value.

Moreover, this [is storing a wrong](https://github.com/UMAprotocol/protocol/blob/5ac8fe8f219271f251d944a4755acf14106827cb/packages/core/contracts/optimistic-assertor/implementation/OptimisticAssertor.sol#L175) value sometimes if `assertion.useDisputeResolution=false`

Reply:
fixed in f0b58ae6bb1dfa521e11a06991cd6190685c548c